tests/jobs/localisation/example_case/scripts/init_test_case.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+"""
+Script initialize the test case by creating the grid files, observation files etc
+"""
+
+import math
+import os
+import random
+
+# pylint: disable=import-error
+import xtgeo
+from common_functions import generate_field_and_upscale, settings, write_upscaled_field
+
+# pylint: disable=too-many-arguments,invalid-name,missing-function-docstring
+# pylint: disable=too-many-locals,redefined-outer-name
```

Comment:
Not necessary to ignore `redefined-outer-name` here I think.

Reply:
Need it after my last updated